### DISCUSSION TITLE
Add CN to extras tab

### DISCUSSION THREAD

**Discussion by darus90** [Mikubill/sd-webui-controlnet#1843] [Ideas]
please add controlnet to extras/automatic1111 tab I am looking for how to apply only the preprocessor to batch images.
TY

**Comment by huchenlei** [comment]
Use ControlNet's `/controlnet/detect` API endpoint please. See https://github.com/Mikubill/sd-webui-controlnet/wiki/API.

**Comment by light-and-ray** [comment]
I've implemented it in an extension!
https://github.com/light-and-ray/sd-webui-cn-in-extras-tab
